worker/pkg/asset/status.go

```diff
@@ -0,0 +1,54 @@
+package asset
+
+import (
+	"strings"
+)
+
+type ArchiveExtractionStatus string
+
+const (
+	ArchiveExtractionStatusPending    ArchiveExtractionStatus = "pending"
+	ArchiveExtractionStatusInProgress ArchiveExtractionStatus = "in_progress"
+	ArchiveExtractionStatusDone       ArchiveExtractionStatus = "done"
+	ArchiveExtractionStatusFailed     ArchiveExtractionStatus = "failed"
+)
+
+func StatusFrom(s string) (ArchiveExtractionStatus, bool) {
```

Comment:
```suggestion
func ArchiveExtractionStatus From(s string) (ArchiveExtractionStatus, bool) {
```

When you rename a struct, don't forget to rename its associated functions!

Reply:
Done!